crates/matrix-sdk-ui/tests/integration/notification_client.rs

```diff
@@ -127,7 +108,6 @@ async fn test_notification_client_with_context() {
         assert_eq!(event.event_type(), TimelineEventType::RoomMessage);
     });
     assert_eq!(item.sender_display_name.as_deref(), Some("John Mastodon"));
-    assert_eq!(item.sender_avatar_url.as_deref(), Some("https://example.org/avatar.jpeg"));
```

Comment:
I had to remove the avatar comparison because the `EventBuilder` expects an `MxcUri` value, but returns a `String`, and I couldn't find a good way to compare them.